### DISCUSSION TITLE
Typo in UI Toolkit Tutorial

### DISCUSSION THREAD

**Discussion by tacman** [Yecats/GameDevTutorials#13] [General]
https://gamedev-resources.com/create-an-in-game-inventory-ui-with-ui-toolkit/

Great tutorial.  Minor issue here with the Location At items reversed:

UI Builder: Visually create and edit your UXML and USS files. Located at Window > UI Toolkit > Debugger.
UI Debugger: A diagnostic tool that lets you traverse the hierarchy of the UI to get useful information around the underlying structure and styling. Located at Window > UI Toolkit > UI Builder.

**Comment by tacman** [comment]
I see it's already been fixed in the .md file here.

**Comment by Yecats** [comment]
So sorry for taking so long to respond! Work has been nuts this week.

Thanks for catching this! I'll get it updated first thing in the morning.
